### DISCUSSION TITLE
add a emoji and unicode inserter tab

### DISCUSSION THREAD

**Discussion by Michaelzhouisnotwhite** [hluk/CopyQ#2745] [Ideas]
may be add a feature to let user conveniently pick emoji or other unicode characters like windows clipboard.

I can create a pull request to implement this feature

**Comment by hluk** [comment]
I would prefer not to have this functionality in the app itself.

Preferably, if there is a command that handles this or just use an external application.

**Reply by Michaelzhouisnotwhite** [reply]
![image](https://github.com/hluk/CopyQ/assets/52443370/1a27bc3d-fcf6-4f23-bb05-f4a4a48cb645)
just like windows. it has a emoji selection area.

**Comment by executed** [comment]
How this can be achieved through CopyQ command?

**Reply by hluk** [reply]
I don't use a command, just a separate tab in CopyQ with few frequently and recently used emojis.
<img width="443" height="629" alt="image" src="https://github.com/user-attachments/assets/0695859c-fddc-4a54-87e4-91c73e28f9f1" />

**Reply by executed** [reply]
@hluk amazing! how do you push those emojis into that specific tab though?
I mean when you copy emoji, let's say from https://emojipedia.org/ - it gets saved by default into your general tab in CopyQ.
Is there any keyboard shortcut to push it to other "emoji" tab, or the only way is to select emoji tab as default one in settings and the roll back when you're done?

**Comment by executed** [comment]
What are possible open source apps that can be used to conveniently search and paste emojis?

**Reply by hluk** [reply]
I think both KDE (`plasma-emojier`) and GNOME have some simple apps for this. There is also web app https://emojipedia.org/.
